content/en/contribute/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Contributing to the Community Health Toolkit"
+linkTitle: "Contributing"
+weight: 10
+description: >
+  How to contribute to the CHT tools and documentation
+---
+
+There are many ways to contribute to the Community Health Toolkit.
+
+**Are You A Developer?**
+
+The [Development](https://forum.communityhealthtoolkit.org/c/developement/7) section of the forum is a great place to introduce yourself and ask questions. Or you can also jump right in:
+- Issues labeled [help wanted](https://github.com/medic/cht-core/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22Help%20wanted%22) are a great place to start. 
+- Learn to [build and host apps]({{% ref "apps" %}}), and share your experiences
+- [Share a new idea or question](https://forum.communityhealthtoolkit.org/) on our Forum
+
+**Are you a researcher, designer, or health practitioner?**
+
+- [Share your expertise](https://forum.communityhealthtoolkit.org) on our Forum
+- Sign up for our [newsletter](https://communityhealthtoolkit.org/contact) to get the latest updates
```

Comment:
And follow us on Twitter, facebook, etc?

Reply:
I was going to add them based on your suggestion, then realized we don't mention them on the main CHT site. We can add them later if they become active.